UM/OutputDevice/OutputDeviceManager.py

```diff
@@ -247,12 +247,10 @@ def removeOutputDevice(self, device_id: str) -> bool:
             self.resetActiveDevice()
         return True
 
-    def addProjectOutputDevice(self, device: "ProjectOutputDevice", add_to_output_devices: bool = False) -> None:
+    def addProjectOutputDevice(self, device: "ProjectOutputDevice") -> None:
```

Comment:
Did/does any other ProjectOuputDevice use this parameter, or was it added originally because of the DF one?

Reply:
Ok, turns out the parameter wasn't present in 4.8 anyway, nothing to see here!